.github/workflows/create-dependabot-pr.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - name: Run dependabot-pr.sh
         run: ./.github/workflows/scripts/dependabot-pr.sh
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
```

Comment:
Do we have this secret set in this repo?  I'm not able to check.

Reply:
It is set at the org level. At least that is what I gather from https://github.com/open-telemetry/community/blob/main/assets.md#opentelemetry-bot